static/skin/taskbar.css

```diff
@@ -123,22 +124,16 @@ label[for=kiwixsearchbox] {
     vertical-align: middle;
 }
 
+a.suggest, a.suggest:visited, a.suggest:hover, a.suggest:active {
+    display: block;
+    text-decoration: none;
+    color: inherit;
+}
```

Comment:
This probably change the color of all the link in the page and we don't want that